lms/djangoapps/shoppingcart/models.py

```diff
@@ -1186,9 +1186,17 @@ def registration_code_used_for_enrollment(cls, course_enrollment):
         has been used during the course enrollment else Returns None.
         """
         try:
```

Comment:
Thanks @afzaledx for pointing that out. Any chance you can commit an update to this? This is a fix to the release candidate and we're trying to get the build out as soon as possible. Thanks.